ch_backup/clickhouse/disks.py

```diff
@@ -142,9 +143,27 @@ def _create_temporary_disk(
         disk_config["endpoint"] = os.path.join(
             f"{endpoint.scheme}://{endpoint_netloc}", source_bucket, source_path, ""
         )
+        disks_config = {tmp_disk_name: disk_config}
+
+        request_timeout_ms = int(disk_config.get("request_timeout_ms", 0))
+        if request_timeout_ms < CH_OBJECT_STORAGE_REQUEST_TIMEOUT_MS:
+            disks_config[tmp_disk_name]["request_timeout_ms"] = str(
+                CH_OBJECT_STORAGE_REQUEST_TIMEOUT_MS
+            )
+            disks_config[disk_name] = {
+                "request_timeout_ms": {
+                    "@replace": "replace",
+                    "#text": str(CH_OBJECT_STORAGE_REQUEST_TIMEOUT_MS),
+                }
```

Comment:
What is the motivation of replacing `request_timeout_ms` for `clickhouse-server` config? AFAIK there is only one long-running operation(`CopyObject`) and is it performing with `clickhouse-disk`. Maybe make sense to keep `request_timeout_ms` only for `clickhouse-disks` config?

Reply:
For consistency.
If source disk is not used in clickhouse-server, better remove additional config for server at all.